tipocket-ctl/tpctl/prepare.py

```diff
@@ -28,6 +28,10 @@
     optgroup.group('Test case build options'),
     optgroup.option('--build-image/--no-build-image', default=False),
 
+    optgroup.group('Test case deploy options'),
+    optgroup.option('--cron', default=False),
```

Comment:
`--cron/--not-cron`

click support boolean option https://click.palletsprojects.com/en/7.x/options/#boolean-flags